apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/ErrorReporting/ErrorReportingExceptionFilter.cs

```diff
@@ -48,38 +51,11 @@ namespace Google.Cloud.Diagnostics.AspNet
     /// </remarks>
     public class ErrorReportingExceptionFilter : IExceptionFilter
     {
-        // The Google Cloud Platform project id as a resource name.
-        private readonly ProjectName _projectResourceName;
-
         // The service context in which this error has occurred.
         // See: https://cloud.google.com/error-reporting/reference/rest/v1beta1/projects.events#ServiceContext
         private readonly ServiceContext _serviceContext;
 
-        private readonly Task<ReportErrorsServiceClient> _clientTask;
```

Comment:
And even if it's not, we should probably encourage creating it synchronously in startup.